server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/Repository.java

```diff
@@ -442,54 +433,32 @@ default Author author() {
         requireNonNull(revision, "revision");
         requireNonNull(query, "query");
 
-        final List<MergeSource> paths = query.mergeSources();
+        final List<MergeSource> mergeSources = query.mergeSources();
         // Only JSON files can currently be merged.
-        paths.forEach(path -> validateJsonFilePath(path.path(), "path"));
+        mergeSources.forEach(path -> validateJsonFilePath(path.path(), "path"));
 
         final Revision normalizedRevision = normalizeNow(revision);
```

Comment:
Just curious. Isn't necessary catching an exception and returning completed future if an exception is caught, like the other code?